Source/DOM classes/Core DOM/CSSStyleDeclaration.m

```diff
@@ -74,7 +74,7 @@ -(NSMutableDictionary *) NSDictionaryFromCSSAttributes: (NSString *)css {
         /* SVG spec specifies some attribute lists might be delimited by white space.
          e.g. stroke-dasharray is 'A list of comma and/or white space separated <length>s'
          http://www.w3.org/TR/SVG/painting.html */
-		if (c == '\n' || c == '\t') {
+		if (isspace(c)) {
```

Comment:
`isspace()` is a C function that determines if a character is of a few selected types. For more information, there is a man page with more info. Just type `man isspace` in a Terminal window.
And yes, `\n` and `\t` are covered.